react/components/SearchBar/components/AutocompleteInput.js

```diff
@@ -94,16 +104,33 @@ const AutocompleteInput = ({
   return (
     <div className={`${handles.autoCompleteOuterContainer} flex`}>
       <div className={classContainer}>
-        <Input
-          ref={inputRef}
-          size="large"
-          value={value}
-          prefix={hasIconLeft && prefix}
-          suffix={suffix}
-          {...restProps}
-          error={Boolean(inputErrorMessage)}
-          errorMessage={inputErrorMessage}
-        />
+        {
+          toggleableInput ?
+            showInput ?
+              <div className={handles.fadeIn}>
+                <Input
+                  ref={inputRef}
+                  size="large"
+                  value={value}
+                  prefix={hasIconLeft && prefix}
+                  suffix={suffix}
+                  {...restProps}
+                  error={Boolean(inputErrorMessage)}
+                  errorMessage={inputErrorMessage}
+                />
+              </div> : suffix
```

Comment:
When the `toggleInput`changes from `true` to `false`, the Input is not rendered anymore.  This way, there is no animation when the user closes the input. 

What do you think about to add an animation in both situations: opening and closing?